linode/manager
# feat: [M3-6751] - Edit Subnet drawer

## Description 📝
Add edit subnet drawer to the VPC details page

## Preview 📷
https://github.com/linode/manager/assets/115299789/23709f21-ebd9-4c9e-b2a0-9acec9f6e761

Restricted user with no permissions
![edit subnet restricted](https://github.com/linode/manager/assets/115299789/0f0b8215-3972-4302-8f4d-d04a3444fcad)

## How to test 🧪
```
yarn test SubnetEditDrawer
```
Go to a VPC's details page and create a subnet if you don't already have one. Then click the edit button in the action menu and edit the subnet's label. Ensure the label updates in the table after saving
